spec/system/home_spec.rb

```diff
@@ -174,4 +174,32 @@ def ie_alert_box_xpath
 
     expect(page).not_to have_css(".title", text: "Featured")
   end
+
+  describe "Header Card" do
+    scenario "if there is header card with link, the link content is rendered" do
+      create(:widget_card, :header, link_text: "Link text", link_url: "consul.dev")
+
+      visit root_path
+
+      expect(page).to have_link "Link text", href: "consul.dev"
+    end
+
+    scenario "if there is header card without link, the link content is not rendered" do
+      create(:widget_card, :header, link_text: nil, link_url: nil)
+
+      visit root_path
+
+      within(".header-card") { expect(page).not_to have_link }
+    end
+
+    scenario "if there is header card without link and with text, the link content is not rendered" do
+      widget = create(:widget_card, :header, link_text_es: "Link text ES", title_es: "Title ES")
+      widget.update!(link_text: "", link_url: "")
+
+      visit root_path
+      select("Español", from: "locale-switcher")
```

Comment:
```suggestion
      visit root_path(locale: :es)
```

This way we avoid an extra request and make the test more concise.